scripts/gitlab/test-linux.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+echo "________Running test-linux.sh________"
+set -e # fail on any error
+set -u # treat unset variables as error
+
+FEATURES="json-tests,ci-skip-tests"
+OPTIONS="--release"
+#use nproc `linux only
+THREADS=$(nproc)
```

Comment:
same here

Reply:
template answer!!

Reply:
we can check it in a separate pipeline @TriplEight